localstripe/resources.py

```diff
@@ -753,6 +754,7 @@ def __init__(self, name=None, description=None, email=None,
                 assert type(data['value']) is str and len(data['value']) > 10
             if payment_method is not None:
                 assert type(payment_method) is str
+            assert type(balance) is int
```

Comment:
Just checking: it cannot be a float, right?

Reply:
I admit I did a very basic implementation, so I didn't check that much. I just checked that with the same steps Stripe gives me ~ the same results.

For example, the customer balance is not updated after invoicing, so it's not a really good implementation after the first invoice.

But for your specific question about the type, I just checked and the doc says it's an integer, so I think we are good for this one:
![2023-06-16-104243_grim](https://github.com/adrienverge/localstripe/assets/10905922/98513a8f-f105-421c-a206-6b3504865846)

Reply:
:+1: